src/update_validation.rs

```diff
@@ -134,7 +134,7 @@ impl UpdateValidation {
         }
 
         self.authenticated = true;
-        info!("update validation status set to \"authenticated\"");
+        debug!("status set to \"authenticated\"");
```

Comment:
please set back to info.
we talked about this as i implemented it. this is not a spamming log, as it is only part of a update validation.

what is your goal here anyway? you do delete the prefix "update validation" here but add it to error logs ("update validation: ").

Reply:
> please set back to info.
we talked about this as i implemented it. this is not a spamming log, as it is only part of a update validation.

part of this PR is a change in handling connection status. there we already have now a info! log:
```
    async fn handle_connection_status(&mut self, auth_status: AuthenticationStatus) -> Result<()> {
        match auth_status {
            AuthenticationStatus::Authenticated => {
                if !self.authenticated_once {
                    info!("Succeeded to connect to iothub");
                    ...
                   self.update_validation.set_authenticated().await?;
```
so the reason to change it was that we would have the log twice.

>what is your goal here anyway? you do delete the prefix "update validation" here but add it to error logs ("update validation: ").

the logs already start with file name, e.g.:
`omnect_device_service::update_validation: no update to be validated`
it looks weired to have this twice at the beginning. for the error log (in a context()) you want to know the origin. that's why I added it there